arc_run_rocrate_galaxy/README.md

```diff
@@ -0,0 +1,81 @@
+---
+authors:
+  - name: Caroline Ott
+    orcid: 0000-0002-6601-2165
+    affiliation: 1
+affiliations:
+  - name: Computational Systems Biology, Rheinland-Pfälzische Technische Universität Kaiserslautern-Landau, Kaiserslautern, Germany
+    index: 1
+---
+
+# ARC Workflow Run RO-Crate Profile integration with Galaxy
+
+## Motivation or Explanation of the Problem
+
+Research Object Crates (RO-Crates), specifically **Workflow RO-Crates** and **Workflow Run RO-Crates**, are essential standards for capturing and exchanging metadata about workflows and their executions. Similarly, [ARCs](https://arc-rdm.org/) use RO-Crate profiles designed to be compatible with the existing [Workflow Run RO-Crate](https://www.researchobject.org/workflow-run-crate/) and [Workflow RO-Crate](https://about.workflowhub.eu/Workflow-RO-Crate/) specifications. This compatibility is ensured when profile recommendations are followed.
+
+One goal is, that ARCs which include [Galaxy](https://galaxyproject.org/) workflows, should be convertable to **Workflow and Workflow Run RO-Crates** that they can be imported and executed by Galaxy. While profile descriptions exist ([see example profile](https://github.com/nfdi4plants/arc-wr-ro-crate-profile/blob/harmonize/profile/arc_wr_ro_crate.md)), practical examples are lacking.
+
+Additional technical challenges include:
+
+* ARC workflow descriptions in the scaffold representation use `.cwl` wrappers for Galaxy workflows. These wrappers are not intended to appear in the resulting RO-Crate, since Galaxy is the relevant workflow engine.
+* The current profiles lack position and prefix information for workflow inputs in their JSON-LD representations.
+* Process sequences in ARCs may reuse inputs also specified as workflow run invocations, so it must be clear which elements in the process sequence also map to workflow inputs.
+
+---
+
+## Proposed Solution with Discussion of Constraints
+
+* To encode position and prefix information for workflow inputs, the `identifier` property of the `FormalParameter` type is extended:
+
+  * The `identifier` is now restricted to a `PropertyValue` with the following properties:
+
+    * `name`: either `Prefix` or `Position`
```

Comment:
Mention potential ontology terms. Start Ontology Term Addition Mechanism in https://github.com/nfdi4plants/nfdi4plants_ontology